tests/e2e/e2e_test.go

```diff
@@ -1121,45 +1118,3 @@ func (s *IntegrationTestSuite) TestGeometricTWAP() {
 	// quote assset supply / base asset supply = 1_000_000 / 2_000_000 = 0.5
 	osmoassert.DecApproxEq(s.T(), sdk.NewDecWithPrec(5, 1), afterSwapTwapBOverA, sdk.NewDecWithPrec(1, 2))
 }
-
-// TestStridePoolMigration tests that Stride's pool migration in v15 completes succesfully.
```

Comment:
Do you think it would be useful to add a commented link to this code here, so that if we ever need to test pool migration logic in e2e we can get to it fast?

Reply:
Hmm I don't think it will be useful because of the `transmigrator` contract coming up. 

Also, if I were to work on migrations again, I would start writing the code from scratch instead of searching for the link in the file.

Let me know if you disagree and if so, where you'd like me to put the link for it to be useful

Reply:
Nope, that sounds good!